lessons/Algebra2/exploring-quadratic-models/langs/en-us/pages/model-speed-v-mpg.adoc

```diff
@@ -0,0 +1,53 @@
+= Modeling Fuel Efficiency v. Speed
+++++
+<style>
+.studentAnswerMedium { min-width: 10em !important; }
+td { padding: 0 !important; }
+body.workbookpage .fitb { padding-top: 0.3rem !important; }
+</style>
+++++
+
+For this page, you'll need to have the @starter-file{alg2-fuel-efficiency} open on your computer. If you haven't already, select **Save a Copy** from the "File" menu to make a copy of the file that's just for you.
+
+@vspace{1ex}
+
+
+**In this exercise you'll have a chance to build and fit more models to the data, now armed with new tools that let you experiment with squares and square roots.** These tools are listed here (as well as in your contracts pages):
+
+@vspace{1ex}
+
+[cols="^1a,^1a", stripes="none", frame="none"]
+|===
+| @math{x^2}
+| @math{\sqrt x}
+
+| @show{(contract "num-sqr" '("Num") "Num" "Consumes a number, and produces the square of that number")}
+| @show{(contract "num-sqrt" '("Num") "Num" "Consumes a number, and produces the square root of that number")}
+|===
+
+@n In the Interactions Area, enter the following expressions, one after the other. Can you predict what the output will be? @show{(code '(sqr 16))}, @show{(code '(sqrt 4))}, @show{(code '(sqr (+ 10 6)))}, @show{(code '(sqrt (* 2 2)))}
+
+== Build a Model from Samples
+
+@n The *vertex form* of a quadratic model is shown below. Fill in the vertex using the most-efficient speed from @printable-exercise{pages/exploring-mpg.adoc}:
+
+@center{@math{f(x) = a(x−}@fitbruby{6em}{}{peak efficiency}@math{)^2+k}}
+
+@vspace{1ex}
+
+@center{__But what values of @math{a} and @math{k} will build the best possible model?__}
+
+@n In the Definitions Area, you'll find a quadratic model `g` defined on line 25. Fit this model to your dataset, using `fit-model`. What @math{R^2} value did you get? @fitb{}{}
+
+@n In model `g` the value of @math{a} is @fitb{5em}{@ifsoln{1}}, the value of @math{h} is @fitb{5em}{@ifsoln{0}}, and the value of @math{k} is @fitb{5em}{@ifsoln{0}}.
+
+@n Experiment with different values of @math{a}. Which one draws a parabola that __most closely matches__ the shape of the dots above it? @fitb{10em}{@ifsoln{~-1/100}}. What @math{R^2} value did you get? @fitb{}{@ifsoln{~-127.455}}
+
+@vspace{1ex}
+
+@n Experiment with different values of @math{k}. Which one draws in a parabola that _most closely fits the shape of the data?__ @fitb{4em}{@ifsoln{~33}}. What @math{R^2} value did you get? @fitb{}{@ifsoln{~0.733}}
```

Comment:
@flannery-denny It would definitely be easier, but I'm not convinced that "easy" is optimal here. Giving students [this desmos activity](https://www.desmos.com/calculator/lpt8nrg1jb) would allow them to **complete the worksheet without ever thinking about what these coefficients mean.**

I think there's a real benefit to making the cost of each iteration nonzero. It forces students to actually think about what these coefficients mean before trying another value.